packages/checkout/sdk/src/config/remoteConfigFetcher.ts

```diff
@@ -77,6 +77,9 @@ export class RemoteConfigFetcher {
     // Ensure that the configuration is valid
     this.configCache = this.parseResponse<RemoteConfiguration>(response);
 
+    // eslint-disable-next-line no-console
+    if (this.configCache && this.configCache.connect.walletConnect) console.warn('WalletConnect is enabled');
```

Comment:
Do you think we need this warning here? I was imagining the warning would be useful if they have configured the widgets options with walletConnect but then our remote feature flag is disabled.. we can then warn them `WalletConnect is currently unavailable.` or something along those lines.

Reply:
Ohh right - I misunderstood this part. will add this to the other ticket